components/UWAPlantMap.vue

```diff
@@ -49,6 +75,19 @@ export default {
     markers() {
       // return the array
       return [...plants]
```

Comment:
just return `plants`, no need to return an array with plants spread

Reply:
This was here long before i started on the project. I'll check to see if it's necessary.